ksql-cli/src/main/java/io/confluent/ksql/cli/console/table/builder/KafkaTopicsListTableBuilder.java

```diff
@@ -34,6 +34,7 @@
 
     @Override
     public Table buildTable(final KafkaTopicsList entity) {
+      System.out.println(entity.getTopics());
```

Comment:
Is this line left by accident? Does it need to print something when building the table?